cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_test.go

```diff
@@ -2603,28 +2612,24 @@ func TestCreateInstalledPackage(t *testing.T) {
 					Values: []packagingv1alpha1.PackageInstallValues{{
 						SecretRef: &packagingv1alpha1.PackageInstallValuesSecretRef{
 							Name: "my-installation-values",
+							Key:  "values.yaml",
 						},
 					},
 					},
 					Paused:     false,
 					Canceled:   false,
-					SyncPeriod: &metav1.Duration{(time.Second * 30)},
+					SyncPeriod: nil,
 					NoopDelete: false,
 				},
 				Status: packagingv1alpha1.PackageInstallStatus{
 					GenericStatus: kappctrlv1alpha1.GenericStatus{
-						ObservedGeneration: 1,
-						Conditions: []kappctrlv1alpha1.AppCondition{{
-							Type:    kappctrlv1alpha1.ReconcileSucceeded,
-							Status:  k8scorev1.ConditionTrue,
-							Reason:  "baz",
-							Message: "qux",
-						}},
-						FriendlyDescription: "foo",
-						UsefulErrorMessage:  "Deployed",
+						ObservedGeneration:  0,
+						Conditions:          nil,
+						FriendlyDescription: "",
+						UsefulErrorMessage:  "",
 					},
-					Version:              "1.2.3",
-					LastAttemptedVersion: "1.2.3",
+					Version:              "",
+					LastAttemptedVersion: "",
```

Comment:
Minor changes in the test cases to match the actual `PackageInstall` being created.